tasks.py

```diff
@@ -285,8 +285,9 @@ def bgp_dev_env():
         f.write(bgpd_config)
 
     # Run a BGP router in a container for all of the speakers to peer with.
-    run('([ "$(docker ps -a | grep frr)" ] && docker rm -f frr) || /bin/true',
-            echo=True)
+    run('for frr in $(docker ps -a -f name=frr --format {{.Names}}) ; do '
```

Comment:
If you want a follow-up PR, I think something simpler will do it, maybe like: `docker rm $(docker ps -aq -name=frr)`. Even if there are more than one, docker rm takes more than one container id as argument (and ps -aq should print exactly what docke rm needs)